src/views/Home/components/Mission/index.view.tsx

```diff
@@ -1,8 +1,46 @@
 import React from "react";
+import { ReactComponent as LeftCan } from "images/mission-can-left.svg";
+import { ReactComponent as RightCan } from "images/mission-can-right.svg";
+import { ReactComponent as Flower } from "images/mission-flower.svg";
 import "./mission.scss";
 
-const MissionComponent: React.FC = () => {
-  return <div className="Mission">Hello From MissionComponent</div>;
+interface MissionProps {
+  about_text: { [key: string]: string };
+  mission_text: { [key: string]: string };
+}
+
+const MissionComponent: React.FC<MissionProps> = ({
+  about_text,
+  mission_text,
+}: MissionProps) => {
+  return (
+    <div className="Mission">
+      <div className="Mission--text">
+        <div className="Mission__section">
```

Comment:
I think we should have the same styling for both sections to prevent something like this: 

<img src="https://user-images.githubusercontent.com/20177171/101290104-18d2ab80-37b5-11eb-8038-8fa4e314b5eb.png" width="250px" />

Reply:
The sections are slightly different because of the gradient highlight, but I fixed it with `max-width`!

Reply:
Ahh, that's what I thought. I was hoping that we can use the same structure for both sections so we can put the mission statements inside a list and `map` them instead (where only the first section would include the `Mission__highlight` div). But I'm totally fine with what you have now since we only have two sections. 

However, I could be wrong but I don't think the `max-width` in `&--subsection` is applied anywhere? I think a temporary fix is to add a fixed padding size in `&--text` for smaller screens so the entire thing can fit in 320px. The real fix might has something to deal with the flex display which I'm not entirely sure.

Reply:
Ahh, that's what I thought. I was hoping that we can use the same structure for both sections so we can put the mission statements inside a list and `map` them instead (where only the first section would include the `Mission__highlight` div). But I'm totally fine with what you have now since we only have two sections.

However, I could be wrong but I don't think the `max-width` in `&--subsection` is applied anywhere? I think a temporary fix is to add a fixed padding size in `&--text` for smaller screens so the entire thing can fit in 320px. The real fix might has something to deal with the flex display which I'm not entirely sure.

Reply:
Ahh, sorry about that! That's actually cause I called it `sub-container` instead of `subsection`. The most recent push should solve it, but let me know if you see anything else! Thank you for the help!